storage-local/src/main/java/io/kestra/storage/local/LocalStorage.java

```diff
@@ -50,6 +50,17 @@ public InputStream get(String tenantId, URI uri) throws IOException {
         );
     }
 
+    @Override
+    public List<String> filePathsByPrefix(String tenantId, URI prefix) throws IOException {
```

Comment:
I don't understand in what this is different from the existing method that list all files for a namespace?

Reply:
It's recursive. Its goal is to be able to get every file (note the word file, we don't want directories in here) with their path.
Following what you said above, I can definitely return the entire URI if that's preferable and do the change in NamespaceFile service

Reply:
> Following what you said above, I can definitely return the entire URI if that's preferable and do the change in NamespaceFile service

Yes, full URIs please, internal storage always use full URIs